tests/functional/z_cluster/test_scale_mon_pods.py

```
@@ -0,0 +1,387 @@
+            pytest.skip("INVALID: Mon count is already above three.")
+
+        # Sleep for alert generation
+        log.info(f"Waiting {ALERT_GENERATION_WAIT_TIME} seconds for alert to be generated")
+        time.sleep(ALERT_GENERATION_WAIT_TIME)
+
+        api = prometheus.PrometheusAPI(threading_lock=threading_lock)
+        alerts_response = api.get(
+            "alerts", payload={"silenced": False, "inhibited": False}
+        )
+        if not alerts_response.ok:
+            log.error(f"got bad response from Prometheus: {alerts_response.text}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)